src/futils.cpp

```diff
@@ -66,12 +66,12 @@ std::string getEnv(int env_var) {
   if (env_var < envHTTPPOST || env_var > envTIMEOUT) {
     throw std::out_of_range("Unexpected env variable");
   }
-  return getenv(ENVARKEY[env_var]) ? getenv(ENVARKEY[env_var]) : ENVARDEF[env_var];
+  return getenv(ENVARKEY.at(env_var)) ? getenv(ENVARKEY.at(env_var)) : ENVARDEF.at(env_var);
```

Comment:
I do not understand this change. In the previous `if` condition we are already making sure that the integer is in the range of the supported indexes, therefore it would not make sense to replace the `operator[]` by the method `at()`.

Reply:
Removed.